client/src/Organisms/ContestDetails/ContestDetailsToggler/ContestDetailsToggler.js

```diff
@@ -2,12 +2,22 @@ import { GoTriangleDown, GoTriangleUp } from 'react-icons/go';
 
 import ContestDetailsTogglerStyled from './ContestDetailsTogglerStyled';
 import PropTypes from 'prop-types';
+import useWindowSize from '../../../Hooks/useWindowSize';
+import { useEffect, useState } from 'react';
 
 const ContestDetailsToggler = ({ onClick, toggle }) => {
+  const [visibilityOfIcon, setVisibilityOfIcon] = useState(true);
+  const { width } = useWindowSize();
+
+  useEffect(() => {
+    width >= 1024 ? setVisibilityOfIcon(false) : setVisibilityOfIcon(true);
+  }, [width]);
+
   return (
     <ContestDetailsTogglerStyled onClick={onClick}>
```

Comment:
You could also use media queries to "disappear" the toggler above the breakpoint